packages/ui/react-ui-navtree/src/types.ts

```diff
@@ -6,7 +6,7 @@ import { type Node, type Action } from '@dxos/app-graph';
 
 export type TreeNodeAction = Pick<Action, 'id' | 'label' | 'invoke' | 'actions' | 'properties' | 'icon' | 'keyBinding'>;
 
-export type TreeNode = Pick<Node, 'id' | 'label' | 'properties' | 'icon'> & {
+export type TreeNode = Pick<Node, 'id' | 'label' | 'properties' | 'icon' | 'childrenMap'> & {
```

Comment:
might children map need to be redefined like `children` to avoid it being recursive with `Node` instead of `TreeNode`?

Reply:
Fixed, just added this to test, now removed.